pootle/apps/pootle_translationproject/migrations/0004_rename_templates.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from pootle_translationproject.utils import TPTool
+
+
+def rename_templates(apps, schema_editor):
+    projects = apps.get_model("pootle_project.Project").objects.all()
+    TP = apps.get_model("pootle_translationproject.TranslationProject")
+    Language = apps.get_model("pootle_language.Language")
+    to_update = None
+    for project in projects:
+        source_lang = project.source_language
+        try:
+            templates = project.translationproject_set.get(
+                language__code="templates")
+            # there is a tp called "templates"
+            # - dont do anything
+            continue
+        except TP.DoesNotExist:
+            pass
```

Comment:
Instead of using `try` clauses, why not use `exists` on the queryset? IMHO it would be cleaner code.